dao-utils/src/parts/collection.rs

```diff
@@ -46,17 +47,16 @@ impl<T: ToBytes + FromBytes + CLTyped + Default + PartialEq + Debug + Hash> Orde
     }
 
     pub fn size(&self) -> u32 {
-        self.length.get()
+        self.length.get().unwrap_or(0)
     }
 
     fn move_item(&mut self, from: u32, to: u32) {
-        let value = self.values.get(from).unwrap();
-        self.values.set(to, value);
-        self.values.unset(from);
+        let value = self.values.get(from);
+        self.values.set(to, value.unwrap_or_revert());
```

Comment:
Why did you remove `self.values.unset(from)`? If I'm right, without it, we've got two occurences of the `from` value.

Reply:
Don't be so interested or you'll get catface :cat: